sql/parser/eval.go

```diff
@@ -356,6 +356,14 @@ func EvalExpr(expr Expr, env Env) (Datum, error) {
 		// anywhere else.
 		env = emptyEnv
 	}
+
+	switch t := expr.(type) {
+	case Row:
+		// Row and Tuple are synonmous: convert Row to Tuple to simplify logic
```

Comment:
No, that's not allowed because of the `switch t := expr.(type)` statement. `t` would have to be both type `Row` and type `Tuple` in the same block of code.